tests/index/test_index_document_meilisearch.py

```diff
@@ -197,6 +197,17 @@ def test_add_documents_csv(empty_index, songs_csv):
     assert index.get_primary_key() == "id"
 
 
+def test_update_documents_csv(index_with_documents, small_movies_json_file):
```

Comment:
This test is going to fail because it is using the json file instead of csv. It should be using the `songs_csv` fixture instead of `small_movies_json_file` fixture.